src/cloud/onboarding/onboarding-tasks.md

```diff
@@ -78,9 +92,35 @@ You can submit a password change request using either of the following methods:
 
 -  Locate the welcome email sent to the License Owner email address and follow the link to change your password.
 
--  Copy the store URL from the [Project Web Interface]({{ site.baseurl }}/cloud/project/projects.html) into a browser. Then, append `/admin` to the end of the URL to open the sign in page. Click the **Forgot password?** link to send a password change request to the License Owner email address.
+-  Copy the store URL from the [Project Web Interface][] into a browser. Then, append `/admin` to the end of the URL to open the sign in page. Click the **Forgot password?** link to send a password change request to the License Owner email address.
 
 After you submit the password change request, check your email for the password reset notification. If you do not get the email, check your spam folder.
 
 {:.bs-callout-tip}
-If the password reset fails or you cannot sign in to the Admin panel, a user with admin access can connect to the project using ssh and add a new admin user using the Magento CLI [admin:user:create command]({{ site.baseurl }}/guides/v2.3/install-gde/install/cli/install-cli-subcommands-admin.html#create-or-edit-an-administrator).
+If the password reset fails or you cannot sign in to the Admin panel, a user with admin access can connect to the project using ssh and add a new admin user using the Magento CLI [admin:user:create command][].
+
+## Sign up for Magento status {#status}
+
+Get updates about {{site.data.var.ece}} platform environments and related services from the [Status page][].
+
+The page lists current status for all {{site.data.var.ece}} components and services followed by notifications about incident reports, service upgrades, planned outages, and more. These notifications are ordered by date.
+
+![All status on the page]({{ site.baseurl }}/common/images/cloud/cloud_status.png)
```

Comment:
This graphic is a bit fuzzy.

Reply:
Thanks--they improved status page quite a bit, so needed an updated capture :-)